### PR TITLE
US003 view collection exercise details

### PR DESCRIPTION
**Pull request by andrewmil**
First attempt at an integration test for the collection exercise page.
This test only checks the the survey attributes are presented on the page, there's no checks for event dates or sample upload etc.

There may be conflicts in file naming with [the PR for US002](https://github.com/ONSdigital/ras-integration-tests/pull/5)

[Trello](https://trello.com/c/Gsa6v2ou/37-us003-view-collection-exercise-details-l-5-days)
[Confluence](https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/160235584/US003+View+Collection+Exercise+Details+L)

### CODE REVIEW

**Comment by andrewmil** [comment]
[US003: View Collection Exercise Details (L = 5 days)](https://trello.com/c/Gsa6v2ou/37-us003-view-collection-exercise-details-l-5-days)

File: acceptance_tests/features/pages/collection_exercise.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from acceptance_tests import browser
+from config import Config
+
+
+def go_to(survey, period):
+    browser.visit(f'{Config.RESPONSE_OPERATIONS_UI}/surveys/{survey}/{period}')
```
Comment (rhiGriff) [inline acceptance_tests/features/pages/collection_exercise.py]: This should be consistent with the .format method used in the repo

File: acceptance_tests/features/steps/view_collection_exercise.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+@then('the internal user can view the collection exercise details')
+def internal_user_can_view_bres_2017_collection_exercise_details(_):
+    survey_info = collection_exercise.get_survey_info()
+    assert survey_info == "221 - Business Register and Employment Survey (BRES)"
+    period = collection_exercise.get_period()
+    assert period == "BRES_2017"
+    user_description = collection_exercise.get_user_description()
+    assert user_description == "August 2017"
```
Comment (rhiGriff) [inline acceptance_tests/features/steps/view_collection_exercise.py]: No new line at end of file

File: acceptance_tests/features/pages/collection_exercise.py (hunk 1 above)
Comment (jcox-dev) [reply acceptance_tests/features/pages/collection_exercise.py]: Arguably, an f-string is much more readable than `.format` here (and for URI building in general).

File: acceptance_tests/features/steps/view_collection_exercise_details.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+
+@given('the 2017 collection exercise for the BRES survey has been created')
+def bres_2017_exists(_):
+    pass
+
+
+@when('the internal user navigates to the collection exercise details page')
+def internal_user_views_2017_bres_collection_exercise(_):
+    collection_exercise_details.go_to('bres', 'BRES_2017')
```
Comment (andrewmil) [inline acceptance_tests/features/steps/view_collection_exercise_details.py]: `BRES_2017` needs replaced with the correct period for BRES

Comment (andrewmil) [reply acceptance_tests/features/steps/view_collection_exercise_details.py]: In the int space this is `221_201712` but I'm not sure if that's what it's supposed to be

**Pull request merged**